Helper/Config.php

```diff
@@ -55,4 +55,19 @@ public function getToken(?int $storeId = null): ?string
             $storeId
         );
     }
+
+    /**
+     * Return if product canonical url configuration is enabled or not
+     *
+     * @param int|null $storeId
+     * @return string|null
+     */
+    public function isUseProductCanonicalUrlEnabled(?int $storeId = null): bool
+    {
+        return $this->scopeConfig->isSetFlag(
+            self::XML_PATH_PRERENDER_USE_PRODUCT_CANONICAL_URL,
+            ScopeInterface::SCOPE_STORE,
```

Comment:
`ScopeInterface::SCOPE_STORES` to avoid this logic
![image](https://github.com/aligent/magento2-prerender-io/assets/37521032/8c359b1b-8cd5-4b9d-a31e-dcab4d8d76ef)